views/layout.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href='https://fonts.googleapis.com/css?family=Poiret+One|Josefin+Sans' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="/css/styles.css">
+  </head>
+  <body>
+    <a href="/"><h1>Home Sweet Home</h1></a>
+    <nav>
+      <a href="/apartments"><h3>Apartments</h3></a><a href="/tenants"><h3>Tenants</h3></a>
```

Comment:
consider only showing link to tenants once you know which apartment you're working with. This will make it easier to select tenants where the apartment id is specified